src/modules/users/publishers/PermissionPublisher.spec.ts

```diff
@@ -26,11 +27,11 @@ describe('PermissionPublisher', () => {
     beforeAll(async () => {
       app = await getApp(
         {
-          queuesEnabled: true,
+          queuesEnabled: [PermissionPublisher.QUEUE_NAME],
         },
         {
           consumerErrorResolver: asClass(FakeConsumerErrorResolver, SINGLETON_CONFIG),
-          permissionConsumer: asClass(FakeConsumer, {
+          permissionConsumer: asClass(FakeConsumer as unknown as Constructor<PermissionConsumer>, {
```

Comment:
Could we use the `asMockClass` (not sure about the naming)? just asking because the casting is weird 😅

Reply:
ah, in the end forgot to use it :D